### PR TITLE
fix take + time term as obj (#222)

### PR DESCRIPTION
**Pull request by jl908069**

### CODE REVIEW

**Comment by nschneid** [comment]
I think this is because technically I created a new commit when I did the merge, even though it was based on yours. Try `git pull`, which will probably trigger an automatic merge to your local branch.

**Comment by nschneid** [comment]
And if that fails try `git merge upstream/dev`

**Comment by jl908069** [comment]
OK. But where should I "git pull"? After "git push", I tried "git pull" and it says "Already up to date"
And then I tried "git merge upstream/dev", it returns "merge: upstream/dev - not something we can merge"

**Comment by nschneid** [comment]
See https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork and https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

File: en_ewt-ud-train.conllu [hunk 1]
```diff
@@ -45522,10 +45522,10 @@
 7	since	since	SCONJ	IN	_	9	mark	9:mark	_
 8	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	9	nsubj	9:nsubj	_
 9	took	take	VERB	VBD	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin	3	advcl	3:advcl:since	_
-10	Ric	Ric	PROPN	NNP	Number=Sing	9	obj	9:obj	_
+10	Ric	Ric	PROPN	NNP	Number=Sing	9	iobj	9:iobj	_
 11	there	there	ADV	RB	PronType=Dem	9	advmod	9:advmod	_
 12	last	last	ADJ	JJ	Degree=Pos	13	amod	13:amod	_
-13	year	year	NOUN	NN	Number=Sing	9	obl:tmod	9:obl:tmod	_
+13	year	year	NOUN	NN	Number=Sing	9	obj	9:obj	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:45528]: Actually this should remain obl:tmod. This is not the take-time sense.

Comment (nschneid) [reply en_ewt-ud-train.conllu:45528]: reverted in b9b601c

**Pull request merged**